### ISSUE TITLE
Accessibility: Can't configure Date Range Picker controls as required

### ISSUE THREAD

**Issue by ThomasOrtiz** [blackbaud/skyux-datetime#250]
### Expected behavior
I should be able to mark each control of the [Date Range Picker](https://developer.blackbaud.com/skyux-datetime/docs/date-range-picker?docs-active-tab=design) as required.

### Actual behavior
- The component does not allow consumers to define label elements (to attach `sky-control-label-required`) or has `Required` inputs for any of the Date Range Picker's form controls (Option, From date, To date).
- Example: 
  - Design wants to mark the **Date Option Control** and **From Date Control** (when it shows) as required but leave the **To Date Control** as optional. 

### Steps to reproduce
1. Try to add a date picker with a `sky-control-label-required` label. 

### Severity
- Without exposing the <label> or an input which sets required/aria-required attributes it can fail accessibility best practices ([link 1](https://www.w3.org/TR/WCAG20-TECHS/H90.html), [link 2](https://www.w3.org/TR/WCAG20-TECHS/ARIA2.html)) around denoting required fields

**Comment by ThomasOrtiz** [comment]
Ill make an ADO bug for this sorry

**Comment by ThomasOrtiz** [comment]
Filed on ADO so I will close this issue here.
